go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -305,8 +324,11 @@ func newSharedGatewayOpenFlowManager(patchPort, macAddress, gwBridge, gwIntf str
 		dftFlows = append(dftFlows,
 			fmt.Sprintf("cookie=%s, priority=50, in_port=%s, ipv6, "+
 				"actions=ct(zone=%d, table=1)", defaultOpenFlowCookie, ofportPhys, config.Default.ConntrackZone))
+		exGWBridgeDftFlows = append(exGWBridgeDftFlows,
```

Comment:
it would be nice to build the exGwBridgeDftFlows and dftFlows in 2 separate functions for readability. Maybe in the doctext for the function provide the gist of the tables and what they are used for.

Reply:
Yes I was a bit torn about this. On one hand we keep the functions slimmer, on the other hand it helps (or, it helped me) seeing the "regular bridge conterpart" to understand where to add the flows.
I'll change that.